src/ProcessManager/ChromeManager.php

```diff
@@ -53,6 +53,7 @@ public function start(): WebDriver
         if ($this->arguments) {
             $chromeOptions = new ChromeOptions();
             $chromeOptions->addArguments($this->arguments);
+            $chromeOptions->setBinary(getenv('GOOGLE_CHROME_SHIM') ?: '');
```

Comment:
Can you allow to set this value with a new optional constructor parameter that will fallback to the environment variable please?

Reply:
Can you also use `$_SERVER['GOOGLE_CHROME_SHIM']` instead of `getenv`?

Reply:
@Hakadel ping

Reply:
Sorry I was busy :x Do you think I should use $_SERVER['PANTHER_CHROME_BINARY'] instead of getenv ??? @Gemorroj @stof 

I'm gonna rebase soon as well.